components/SpotCard.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import Image from "next/image";
+import placeholderSpotImage from "../assets/images/card-image-placeholder.jpg";
+
+export default function SpotCard(spot) {
+  return (
+    <>
+      <Card>
+        <SpotImage src={placeholderSpotImage} alt="Placeholder Spot Image" />
+        <SpotInfos>
+          <h3>{spot.spot.category}</h3>
+          <h4>"{spot.spot.name}"</h4>
+        </SpotInfos>
+      </Card>
+    </>
```

Comment:
- **Good news:** You don't need the react fragments `<> </>` because `Cards` groups children: <SpotImage> and <SpotInfos>.
- When you change from the placeholder image to real images, I would recommend that you describe the _alt="text"_ more descriptive. What the image shows/indicates? - but that's a task for the future Feline 🪐

```suggestion
      <Card>
        <SpotImage src={placeholderSpotImage} alt="Placeholder Spot Image" />
        <SpotInfos>
          <h3>{spot.spot.category}</h3>
          <h4>"{spot.spot.name}"</h4>
        </SpotInfos>
      </Card>
```